specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/capacity.json

```diff
@@ -0,0 +1,180 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerRegistryManagementClient",
+    "description": "Admin Container Registry Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/capacity/{capacityName}": {
+      "get": {
+        "x-ms-examples": {
+          "Returns container registry capacity properties.": {
+            "$ref": "./examples/Configuration/GetCapacity.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryCapacity"
+        ],
+        "description": "Returns container registry capacity property.",
+        "operationId": "ContainerRegistryCapacity_Get",
```

Comment:
You probably don't want ContainerRegistry as a prefix to all of your operation groups, unless there is some other kind of Capacity in this RP, Capacity_Get should suffice

Reply:
It is ok, we don't want to collide with Storage Api for example which also has similar operations.

Reply:
You won't collide with storage api - this only has to do with operation groups in this client, it really does negatively impact the usability of generated clients.

Reply:
not addressed

Reply:
i made the change, but for some reason it generates the ps cmdlet with very generis name like Get_AzsCapacity.ps1 what is not appropriate for our needs, we need more specific name to not confuse operator, like: Get-AzsContainerRegistryCapacity.ps1
I will kept this as is for now.